src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/steps/DeletedAccountsGetApiSteps.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.laubackend.idam.serenityfunctionaltests.steps;
+
+import net.thucydides.core.annotations.Step;
+import org.json.JSONException;
+import uk.gov.hmcts.reform.laubackend.idam.serenityfunctionaltests.model.DeletedAccountsRequest;
+import uk.gov.hmcts.reform.laubackend.idam.serenityfunctionaltests.model.DeletedAccountSearch;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DeletedAccountsGetApiSteps extends BaseSteps {
+
+    @Step("Given invalid GET body is generated")
+    public DeletedAccountsRequest generateInvalidDeletedAccountsSearchRequest() {
+        DeletedAccountSearch deletedAccountSearch = DeletedAccountSearch.builder()
+            .userId("user id")
+            .emailAddress("")
+            .firstName("John")
+            .lastName("Smith")
+            .endTimestamp("2023-05-23T09:34:23.432Z")
+            .build();
+
+        DeletedAccountsRequest request = new DeletedAccountsRequest();
+        request.setDeletionSearchLogs(Arrays.asList(deletedAccountSearch));
+        return request;
+    }
+
+    @Step("When valid params are supplied for Get Logon API")
+    public Map<String, String> givenValidParamsAreSuppliedForGetLogonApi() {
+        HashMap<String, String> queryParamMap = new HashMap<>();
```

Comment:
It doesn't matter much here, but generally you want to use `Map` instead of concrete implementation `HashMap`, so this should be `Map<String, String> queryParamMap = new HashMap<>()`. Usually this get marked as code smell (I think) in sonarqube and similar code analysis tools.